examples/discovery-search-app/src/App.js

```diff
@@ -5,6 +5,8 @@ import './app.scss';
 import { settings } from 'carbon-components';
 import { Button } from 'carbon-components-react';
 import Close from '@carbon/icons-react/lib/close/16';
+import { Tabs, Tab } from 'carbon-components-react';
```

Comment:
can you move this to be combined with `import { Button } from 'carbon-components-react';` above?